tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -324,12 +324,12 @@ public static class Tags
         /// <summary>
         /// The queue name associated with the AWS SDK span.
         /// </summary>
-        internal const string AwsQueueName = "aws.queue.name";
+        internal const string AwsQueueName = "messaging.destination";
```

Comment:
Could you also add this change and the changes later in this file to the PR description?

Reply:
Done thx